guide/layouts/partials/example.slim

```diff
@@ -22,18 +22,18 @@ figure.app-example
             | Data
 
     .fr-tabs__panel.fr-tabs__panel--selected tabindex="0" id=%(input-slim-#{anchor_id(caption)}) role="tabpanel" aria-labelledby=%(input-slim-#{anchor_id(caption)})
-      pre
+      pre.highlight
         code.highlight.language-slim
```

Comment:
I'm guessing the `highlight` class on the `code` tag is useless?

Reply:
vais regarder pas bête

Reply:
done ✅